imports/lib/schemas/validators.ts

```diff
@@ -0,0 +1,16 @@
+import { SchemaDefinition } from 'simpl-schema';
+
+const ValidUrl: SchemaDefinition['custom'] = function () {
+  // Tests if a URL is valid by attempting to construct a URL object from it.
+  if (!this.isSet) return undefined;
+  try {
+    // eslint-disable-next-line no-new
```

Comment:
Aside: I'm surprised there's not a `(void)new URL` override or something similar for this /shrug

Reply:
Looks like I can spell this

```
void (new URL(...))
```